src/main/java/org/jabref/logic/importer/fetcher/DoiResolution.java

```diff
@@ -121,6 +121,6 @@
 
     @Override
     public TrustLevel getTrustLevel() {
-        return TrustLevel.SOURCE;
+        return TrustLevel.META_SEARCH;
```

Comment:
This decreases the result quality of the DOI fetcher (always leading to the "right" paper) to the quality of Google Scholar. (From the highest to the lowest)

Can the solution of the title? 😇 

> Make the DOI Resolution Fetcher return nothing when the DOI leads to a host for which a tailored fetcher exists